options/locale/locale_de-DE.ini

```diff
@@ -572,7 +572,7 @@ quick_guide=Kurzanleitung
 clone_this_repo=Dieses Repository klonen
 create_new_repo_command=Erstelle ein neues Repository von der Kommandozeile aus
 push_exist_repo=Bestehendes Repository via Kommandozeile pushen
-bare_message=Dieses Repository hat noch keinen Inhalt.
+empty_message=Dieses Repository hat noch keinen Inhalt.
```

Comment:
The changes in non-english language translation files will be overwritten by crowdin when this gets merged to master. My preference would be not to have the changes, however it's not blocking and so it's up to you if you have time.

Reply:
I think it's helpful to have if just to hint what needs to change on crowdin.